include/ur_api.h

```diff
@@ -9580,6 +9582,203 @@ urEnqueueNativeCommandExp(
                                                                    ///< been enqueued in nativeEnqueueFunc.
 );
 
+#if !defined(__GNUC__)
+#pragma endregion
+#endif
+// Intel 'oneAPI' Unified Runtime Experimental API for enqueuing work through native APIs
```

Comment:
Is this the right comment? Looks like a copy :spaghetti:

Reply:
Oops this is copy pasta. Thanks